Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -93,3 +93,72 @@ public extension Message {
   }
 }
 
+
+public extension Array where Iterator.Element: Message {
```

Comment:
Re: [this](https://github.com/apple/swift-protobuf/pull/496#discussion_r113752518), I'd like to hear a little more of what you're thinking there.